src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const direct of actions) {
+    switch (direct.type) {
+      case 'addProperties':
+        Object.assign(state, direct.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of direct.keysToRemove) {
+          if (key in state) {
```

Comment:
i don`t think , this condition makes some sens, mb remove it

Reply:
Actually it makes sense, he checks if key from action.keysToRemove is in the `state`, so an unnecessary delete statement won't happen if there is no such key